packages/outline-docs/src/guides/component-development/06-properties.mdx

```diff
@@ -21,27 +21,34 @@ import '@phase2/outline-container';
   bottom-margin="spacing-8"
 >
   ## Description
-  Lit components receive input and store their state as JavaScript class fields or properties.
+  Outline components receive input and store their state as JavaScript class fields(normally called attributes) or properties.
   
   [Reactive properties](https://lit.dev/docs/components/properties/) are properties that can trigger the reactive update cycle when changed, re-rendering the component, and optionally be read or written to attributes.
 
   ## Documentation Status
-  <outline-alert status="warning" size="large">
-    <span slot="header">Status: Needs Work/In Progress</span>
+  <outline-alert status="success" size="large">
+    <span slot="header">Status: Complete / Update to data May 2023</span>
     <p>
       This documentation is in need of additional work, or is currently in progress.
     </p>
   </outline-alert>
 </outline-container>
 
 ### Property usage basics
-
 - Properties should be commented using [JSDoc](https://github.com/jsdoc/jsdoc).
 - Each property should provide a comment including a description of the attribute functionality or purpose.
 - Pass to `type` the appropriate [converter](https://lit.dev/docs/components/properties/#conversion-type) (`String`, `Boolean`, etc.).
 - Pass to `attribute` an attribute name in kebab-case.
 - Attribute names must not collide with HTML standard ones `title`, `href`, etc.
   - This means it is best to name properties more like `link-title`, `link-href`, etc.
+- Lit [documentation]()
+
+### Slots versus properties
+- When do I use a property? 
+  - Properties are used for non-content features that impact the state of the component. For example, active versus inactive states.
+- The component shouldn't change its own public properties, except in response to user input. For example, you could set a property to change background color, but the component itself shouldn't change it's own color.
+- Slots are used for content, such as HTML markup, text, icons, or images. 
```

Comment:
Related to the above, I think this statement goes to far and also missed in my mind the actual primary use case for slots: flexible regions of a component

Icons in particular it really depends on whether the icons are built into the Outline package itself and thus are set using the name of the icon as a property on a component or if it is an svg upload that the content team does using any svg they wish.

Text often needs to be used as alt text or other things requiring attributes inside of the component, and thus cannot always use slots either.

Reply:
Perhaps clarification can be given by some simple scenarios like:

For Example:
- "if you were building Web Components to be consumed by a React App that does X, Y, and Z, you might do it this way.
- "if you were building Web Components to be consumed by a Drupal App that does X, Y, and Z, you might do it this way.

I feel like that's probably where some of the assumptions are coming from. Our typical work still being Drupal, we need to account in our docs (and sample integrations would be nice) how to do things with different consumers.